modules/atomic-widgets/prop-types/primitives/string-prop-type.php

```diff
@@ -65,4 +65,8 @@ private function validate_enum( $value ): bool {
 	private function validate_regex( $value ): bool {
 		return preg_match( $this->settings['regex'], $value );
 	}
+
+	protected function sanitize_value( $value ) {
+		return ! empty( $value ) ? sanitize_text_field( $value ) : null;
```

Comment:
maybe we should keep the `wp_kses_post`? not sure whats better

Reply:
The `sanitize_text_field` method removes *any* HTML element, this is why I'm currently testing the use of 
```
<b>...</b>
```
`wp_kses_post` on the other hand would remove only the dissallowed tags, such as a 
```
<script />
```

Reply:
OK, so I guess `sanitize_text_field` is better